modules/exasock/exasock-tcp.c

```diff
@@ -1426,8 +1426,10 @@ static void exasock_tcp_dead(struct kref *ref)
 /* need biglock for this one function to avoid race condition */
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2, 6, 36)
 static DECLARE_MUTEX(update_biglock);
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(6, 6, 0)
```

Comment:
This change actually showed up in 6.4.

```suggestion
#elif LINUX_VERSION_CODE < KERNEL_VERSION(6, 4, 0)
```